Cargo.toml

```diff
@@ -45,12 +49,20 @@ bytes = "1.5.0"
 axum-core = { version = "0.4.3", optional = true }
 http = { version = "1", optional = true }
 async-trait = { version = "0.1", optional = true }
+tokio-util = { version = "0.7", features = ["codec", "io"] }
+futures = { version = "0.3", default-features = false, features = ["std"] }
 
 [features]
 default = ["simd"]
 simd = ["simdutf8/aarch64_neon"]
-upgrade = ["hyper", "pin-project", "base64", "sha1", "hyper-util", "http-body-util"]
-unstable-split = []
```

Comment:
Is this intentional to remove `unstable-split`? I would like to keep it

Reply:
Sure, I can put it back in

Reply:
well, the question is, why is it unstable?